### PR TITLE
Snowflake batch prediction api

### PR DESCRIPTION
**Pull request by sickmint79**
support for new community article

### CODE REVIEW

File: databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query_s3.ipynb [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+    "ExecuteTime": {
+     "end_time": "2020-03-22T08:20:24.458764Z",
+     "start_time": "2020-03-22T08:20:16.578812Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<snowflake.connector.cursor.SnowflakeCursor at 0x12326dd90>,\n",
```
Comment (zmarkan) [inline databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query_s3.ipynb]: Not sure what these SnowflakeCursors are?

File: databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query.ipynb [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+     "end_time": "2020-03-21T08:46:42.658920Z",
+     "start_time": "2020-03-21T08:46:40.903948Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "queued batch job: https://app.datarobot.com/api/v2/batchPredictions/1234567891234567890/\n"
```
Comment (zmarkan) [inline databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query.ipynb]: This looks like a scrambled number, right?

File: databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query_s3.ipynb (hunk 1 above)
Comment (sickmint79) [reply databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query_s3.ipynb]: i assume some local hex memory address or something.

File: databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query.ipynb (hunk 2 above)
Comment (sickmint79) [reply databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query.ipynb]: it's fake

**Pull request merged**